docs/rocprof.md

```diff
@@ -1,12 +1,18 @@
 # rocprof
+
 ## 1. Overview
+
 The rocProf is a command line tool implemented on the top of  rocProfiler and rocTracer APIs. Source code for rocProf may be found here: 
-GitHub: https://github.com/ROCm-Developer-Tools/rocprofiler/blob/amd-master/bin/rocprof  
+GitHub: https://github.com/ROCm-Developer-Tools/rocprofiler  
 This command line tool is implemented as a script which is setting up the environment for attaching the profiler and then run the provided application command line. The tool uses two profiling plugins loaded by ROC runtime and based on rocProfiler and rocTracer for collecting metrics/counters, HW traces and runtime API/activity traces. The tool consumes an input XML or text file with counters list or trace parameters and provides output profiling data and statistics in various formats as text, CSV and JSON traces. Google Chrome tracing can be used to visualize the JSON traces with runtime API/activity timelines and per kernel counters data.
+
 ## 2. Profiling Modes
-‘rocprof’ can be used for GPU profiling using HW counters and application tracing
+
+‘rocprof’ can be used for G5PU profiling using HW counters and application tracing
```

Comment:
Please correct typo. should be GPU profiling

Reply:
Fixed in latest